addons/pos_restaurant/static/src/js/floors.js

```diff
@@ -259,7 +259,8 @@ var TableWidget = PosBaseWidget.extend({
                         args: [[['id', '=', table_id]], fields],
                         limit: 1,
                     })
-                    .then(function (table){
+                    .then(function (result){
+                        var table = result[0];
```

Comment:
I don't get it. there is a `limit: 1` on the RPC, so what does this do?

Reply:
Maybe there was a change in the rpc at some point of time? The result however is an array, but maybe when the code was written it only returned the object. ref: https://github.com/odoo/odoo/issues/27921
![selection_110](https://user-images.githubusercontent.com/40355961/47147938-9f735880-d2cf-11e8-84a5-c14a6f0f82e0.png)